tests/binary_math/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -17,9 +17,9 @@ var from = offset > left ? 0 : (left - offset) >> level;
 var to = ((right - offset) >> level) + 1;
 if (rawIndex < 1 << (list._level + SHIFT)) {
 }
-var res = size < SIZE ? 0 : (size - 1) >>> SHIFT << SHIFT;
+var res = size < SIZE ? 0 : ((size - 1) >>> SHIFT) << SHIFT;
 sign = 1 - 2 * (b[3] >> 7);
 exponent = (((b[3] << 1) & 0xff) | (b[2] >> 7)) - 127;
-mantissa = ((b[2] & 0x7f) << 16) | (b[1] << 8) | b[0];
+mantissa = (((b[2] & 0x7f) << 16) | (b[1] << 8)) | b[0];
```

Comment:
This one looks a _little_ excessive but it is adding parens from `x | y | z` to `(x | y) | z`.